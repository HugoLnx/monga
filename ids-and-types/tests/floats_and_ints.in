int main() {
  float x, h;
  char y;
  h = 3.0;
  x = h;
}
