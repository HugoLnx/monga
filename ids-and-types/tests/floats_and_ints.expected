declarations:
  declaration:
    function:main,259,0
      block:
        var_declarations:
          variables:
            variable:x,261,0
            variable:h,261,0
          variables:
            variable:y,260,0
        statements:
          statement:
            attribution:
              var:h
              exp:3,3.00000
          statement:
            attribution:
              var:x
              exp:5
                var:h
